.github/workflows/test.yml

```diff
@@ -63,6 +63,23 @@ jobs:
           # Add HTTPS remote.
           lxc remote add localhost "$(lxc config trust add --name lxd-terraform-provider --quiet)"
 
+      - name: Configure MinIO
+        run: |
+          arch=$(dpkg --print-architecture)
+          mkdir -p /opt/minio
```

Comment:
It's surprising (to me) that `/opt` is writable by the runner user. Maybe a special thing on GHA runners... This is normally root writable only.

Doesn't matter, just surprising :)